### ISSUE TITLE
Error: Failed to sign request: Request failed with status code 429

### ISSUE THREAD

**Issue by vioan** [zerodytrash/TikTok-Chat-Reader#17]
Since a few hours, the live demo is not working anymore, I get

Error: Failed to sign request: Request failed with status code 429

Is there anything can be done to fix it?

Thanks a lot!

**Comment by zerodytrash** [comment]
TikTok has banned the server IP due to too many requests. I will look into it soon.

**Comment by zerodytrash** [comment]
Should work again.

**Comment by vioan** [comment]
now is giving

You have opened too many connections or made too many connection requests. Please reduce the number of connections/requests or host your own server instance. The connections are limited to avoid that the server IP gets blocked by TokTok.

**Comment by vioan** [comment]
ah, ok, it seems that is because I have 4 tabs opened and they were connections made to the live demo

**Comment by zerodytrash** [comment]
yes, i have limited the number of open connections because some people use the website to make a lot of connections which causes tiktok to block the server ip.

**Comment by Houss3m** [comment]
Is there any way that I can host the server on my own?
